lib/common_filters.ex

```diff
@@ -61,15 +61,20 @@ defmodule EctoShorts.CommonFilters do
     queryable :: Ecto.Queryable.t(),
     params :: Keyword.t | map
   ) :: Ecto.Query.t
-  def convert_params_to_filter(query, params) when params === %{}, do: query
+  def convert_params_to_filter(query, params) when params === %{} do
+    query
+  end
+
   def convert_params_to_filter(query, params) when is_map(params) do
-    convert_params_to_filter(query, Map.to_list(params))
+    params = Map.to_list(params)
```

Comment:
This seems unnecessary, there's nothing wrong with calling functions on params 😅

Reply:
I'll undo, agreed there's nothing wrong with it. just personal preference